src/plugins/workspace/public/components/workspace_form/data_source_connection_table.tsx

```diff
@@ -242,6 +242,7 @@ export const DataSourceConnectionTable = ({
       itemIdToExpandedRowMap={itemIdToExpandedRowMap}
       isExpandable={true}
       selection={selection}
+      className="workspace-detail-direct-query-table"
```

Comment:
Can we add this classname to `src/plugins/workspace/public/components/workspace_form/data_source_connection_table.tsx`. I think this my mistake, removed this class name when refactor data source table in #7961 . I think we need this classname in the workspace create page too.